authors/wubwub/05-SegWit2x/hr_HR/final.md

```diff
@@ -0,0 +1,181 @@
+
+**Vlak** je ovdje **blockchain**, **vagoni** su **blokovi**, **putnici** su **transakcije**, a **prtljaga** su **potpisi**.
+
+Što su ovdje developeri odlučili napraviti? Odlučili su "prepoloviti transakciju" i odvojiti **potpis** od **ostatka transakcije**, i tako su povećali kapacitet blokova koji su 1mb bez povećavanja veličine blokova (blok s potpisima iznosi ~2mb, maksimalno 4mb).
+
+Čvorovi koji žele provjeriti legitimitet transakcija mogu zatražiti i potpise za svaku transakciju, a oni koji ne žele, oni dobivaju samo transakcije.
+
+## NYA dogovor i SegWit2x
+
+[New York Agreement][NYA] je dogovor između nekih od najvećih kompanija u Bitcoin prostoru i velikih rudara. Dogovoreno je da će se prvo aktivirati SegWit, a 90 dana poslije će se aktivirati SegWit2x koji će još podići veličinu bloka na ~4-8mb.
+Cijeli razvoj Segwit2x je nadgledan od strane Jeff Garik-a,
+ 
+*Dodatna zanimljivost je to što nitko od __Core developera__ nije pozvan na NYA dogovor.*
+
+Na slikama možemo vidjeti neke od ključnih zagovornika SegWit2x (Jihan Wu i Roger Ver) kako promoviraju Bitcoin Cash, time je jasno gdje oni stoje i da su zapravo protiv Bitcoina.
+
+![01](../Images/01.png)
+
+![02](../Images/02.png)
+
+Od početnog broja od 58 kompanija u NYA dogovoru, 7 kompanija je odustalo od dogovora, dok je 8 kompanija prešlo na Bitcoin Cash. Ostaje nam 41 kompanija koja još podržava SegWit2x nadogradnju.
+
+[Ovdje možete pratiti popis kompanija][segwitstatus] koje su za ili protiv.
+
+### UAHF i Bitcoin Cash
+
+Nakon svih neuspješnih pokušaja dogovora i dolaska do zajedničkog konsenzusa, Bitmain je odlučio 1.8.2017., ako dođe do UASF (User Activated Soft [Fork]) od strane Core developera, a time posljedično i implementacije SegWita.
+
+Bitmain će pokrenuti svoj UAHF (User Activated Hard Fork). Ovim forkom je nastao Bitcoin Cash, razlika u odnosu na Bitcoin Core je 8mb veličina blokova i "podrška" za ASICBoost.
+
+Kao što već vjerojatno znate, svaka strana je dobila što želi, BItmain je forkao Bitcoin na Bitcoin Cash, a Core developeri su uz konsenzus mreže implementirali SegWit u obliku soft forka (što znači da nije potrebna nikakva intervencija krajnjeg korisnika).
+
+## Segwit2x kao korporativni napad na Bitcoin?
+
+Uz ove gore navedene probleme s SegWit2x nadogradnjom, također postoji problem ne postojanja konsenzusa oko SegWit2x. Naime, sve te kompanije i rudari su potpisali NYA dogovor, no nitko nije pitao Core developere slažu li se s time, no ni Core developeri nisu toliko bitni kao što su bitni sami korisnici Bitcoina.
+
+Ljepota Bitcoina kao decentraliziranog protokola je baš u tome što je demokratski, svatko može glasati tako da kod kuće ima Bitcoin čvor koji primjenjuje stroga pravila po kojima neki blok smatra validnim. Tako da ako rudari aktiviraju SegWit2x i rudare veće blokove, čvorovi ih neće prihvatiti jer po pravilima Bitcoina ti blokovi nisu validni.
```

Comment:
Koji rudari? Oni iza BCH ili drugi? Da li se S2X aplicira i na BCH i na BTC ili samo jedan od njih? Hoće li biti 4 forka nakon svega ovoga, ili 2? Zašto čvorovi ne bi prihvatili odluku rudara? Kakve veze to ima s krajnjim korisnicima ako će dobiti Bitcoin koji je brži i jeftiniji za korištenje? Puno toga visi u zraku tu kad pogledaš s strane neupućenog promatrača.